index.html

```diff
@@ -19,14 +20,20 @@
     <div class="logo">
       <p><a href="./index.html">My Logo</a></p>
     </div>
```

Comment:
- [ ] Kindly make sure your logo text(**"My Logo"**) follows the styling of your chosen Figma template to stick to the Figma design. e.g.: text-decoration, text color once clicked,...

- [ **OPTIONAL** ] For good tags semantics I suggest you remove the `<p>` tag to make your code base looks more professional i.e it can only look like this: 
```
   <div class="logo">
      <a href="./index.html">My Logo</a>
   </div>
```